lib-object-oriented/line.js

```diff
@@ -1,34 +1,77 @@
 import Tag from './tag'
 
 export default class Line {
-  constructor( content ) {
+  constructor({ content, type }) {
     this.content = content
+    this.tagType = type
+    this.symbolCount = 0
+    this.getSymbolCount()
+    this.newLinesToSpaces()
+  }
+
+  newLinesToSpaces() {
+    const tagType = this.tagType,
+          content = this.content,
+          symbolCount = this.symbolCount
+
+    const findNewLine = element => {
```

Comment:
I would consider making this it's own function in the class, just to help clean up this code a little.  I suggest that each function should have only a single purpose, and this adds a purpose into this function.

If you don't want to do that, I would leverage the conciseness of ES6 to reduce the footprint:
```js
const findNewLine = element => element.type === 'new_line'
```
Finally, one of the reasons for declaring the different character types as constants was so that we could re-use those constants without fear of typo throughout our project, so I would actually import the `NEW_LINE` constant from token.js, and use that in the comparison:
```js
const findNewLine = element => element.type === NEW_LINE
```